eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -421,3 +442,4 @@ stages:
               SparseCheckoutPaths:
                 - docs-ref-services/
                 - metadata/
+              DocValidationImageId: "$(DocValidationImageId)"
```

Comment:
This validation is failing probably because the docker image hasn't been fetched during this job.

Reply:
Yeah, fixed the problems. 
1. The command is not correct at the first place (no `-e` before env)
2. The public feed did not config into docker `m2` file. Fixed the docker already
Trigger some other tests right now.

Reply:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1245989&view=logs&j=bb4ed450-a9b2-5e9f-2352-502bec33ec8b&t=f70dc54e-cb2f-5d88-e349-fda3612dab78